content/recipes/serve-static.md

```diff
@@ -35,6 +35,15 @@ export class AppModule {}
 
 With this in place, build the static website and place its content in the location specified by the `rootPath` property.
 
+#### Configuration
+
+[ServeStaticModule](https://github.com/nestjs/serve-static) can be configured with a variety of options to customize its behavior.
+You can set the path to render your static app, specify excluded paths, enable or disable setting Cache-Control response header, etc. See the full list of options [here](https://github.com/nestjs/serve-static/blob/master/lib/interfaces/serve-static-options.interface.ts).
+
+> warning **Notice** The default `renderPath` of the Static App is `*` (all paths), and the module will send "index.html" files in response.
```

Comment:
I'm not sure this is entirely clear.  Could you please try to explain in a bit more detail?  Don't worry too much about the exact wording - I'm glad to help with that.  But if you can explain how this works in conjunction with client side routing, that would be good.

I have played with this module in the past and I agree with what you're attempting to do to make it a bit more clear.  As I re-read this paragraph, I'm not really quite understanding the behavior.  I can go back and experiment to remember more clearly how it works, but I wanted to ask you to attempt to explain it more first, since maybe it's more fresh in your mind.

Reply:
Of course, I'll try to explain, but I am not really sure, that I do understand it clearly (see the [Issue](https://github.com/nestjs/docs.nestjs.com/issues/1153)).

So, what do you think about this one:

Reply:
The works like a middleware, that sends specified index file(s) in response to every request (if the route is not excluded). At the same time, the routes, specified in your controllers have a bigger priority, so they will work before the ServeStaticModule will try to serve static files in response.

That means you can configure Client-side routing and as the user navigates around your SPA no full page reloads happen.

[ServeStaticModule](https://github.com/nestjs/serve-static) can be configured with a variety of options to customize its behavior.
You can set the path to render your static app, specify excluded paths, enable or disable setting Cache-Control response header, etc. See the full list of options [here](https://github.com/nestjs/serve-static/blob/master/lib/interfaces/serve-static-options.interface.ts).

> warning **Notice** The default `renderPath` of the Static App is `*` (all paths), and the module will send "index.html".